kumascript/src/api/page.ts

```diff
@@ -54,6 +62,50 @@ const page = {
   translations(this: KumaThis, path) {
     return this.info.getTranslations(path || this.env.url);
   },
+
+  async badges(this: KumaThis, aPage, spaceAround = false) {
```

Comment:
I have made it return an array, so .ejs files now use `join` functions. Although it required `.then(badges => " " + badges.join(" "))` in order to preserve leading space. I have also extracted the logic of `badges` function to `badges.ts`, so the code looks a bit different than what you have requested.
There is a problem with circular dependency though, because now `badges` require the utility function `hasTag` from `page`. If you like the changes then I think it would be best to move `hasTag` somewhere else, but if you prefer the code that you initially suggested, I can add it instead.